tests/nexus/test_handler.py

```diff
@@ -556,7 +557,7 @@ class NonSerializableOutputFailure(_FailureTestCase):
 )
 @pytest.mark.parametrize("with_service_definition", [True, False])
 async def test_start_operation_happy_path(
-    test_case: Type[_TestCase],
+    test_case: type[_TestCase],
```

Comment:
I'll make the `Type -> type` change globally at some point, hopefully when it's enforced by CI. For now I'm mainly doing it in Nexus code.